.github/workflows/flutter_ci.yml

```diff
@@ -1,6 +1,6 @@
 name: Flutter CI
 
-on: [pull_request]
```

Comment:
please revert this

Reply:
I'm testing Github CI. I will revert and request you to review after finishing